lib/braveAlerter.js

```diff
@@ -28,35 +28,48 @@ class BraveAlerter {
     }
 
     async sendSingleAlert(toPhoneNumber, fromPhoneNumber, message) {
-        const response = await twilio.sendTwilioMessage(
-            toPhoneNumber,
-            fromPhoneNumber,
-            message
-        )
+        let response
+        try {
+            response = await twilio.sendTwilioMessage(
+                toPhoneNumber,
+                fromPhoneNumber,
+                message
+            )
+        } catch (err) {
+            // Error handling will be done below because response is undefined
+            helpers.log(err)
+        }
 
-        if (response !== undefined) {
-            helpers.log(response.sid)
-        } else {
+        if (response === undefined) {
             helpers.log(`Failed to send single alert: ${message}`)
         }
     }
 
     // See README for description of alertInfo
     async startAlertSession(alertInfo) {
-        let isAlertSent = false
-
+        let response
+        
         // Send initial message
         if (alertInfo.toPhoneNumber && alertInfo.fromPhoneNumber) {
-            const response = await twilio.sendTwilioMessage(
-                alertInfo.toPhoneNumber,
-                alertInfo.fromPhoneNumber,
-                alertInfo.message
-            )
-
-            isAlertSent = (response !== undefined)
+            try {
+                response = await twilio.sendTwilioMessage(
+                    alertInfo.toPhoneNumber,
+                    alertInfo.fromPhoneNumber,
+                    alertInfo.message
+                )
+            } catch (err) {
+                // Error handling will be done below because response is undefined
+                helpers.log(err)
+            }
         }
-
-        if (!isAlertSent) {
+            
+        if (response !== undefined) {
```

Comment:
There are other failure cases here based on other Twilio statuses. We can do better error handling based on that.

Reply:
Leaving as is for now since this is no worse than what we are currently doing and hopefully this will be replaced by the app.